upsertGitHubTag/deployment/index.js

```diff
@@ -252,6 +254,27 @@ function processEvent(event, callback) {
         githubEventType +
         " from GitHub.",
     });
+    return;
+  }
+  // If bucket name is not null (had to put this for the integration test)
+  if (process.env.BUCKET_NAME) {
+    // Send payload to s3
+    const uploadDate = new Date();
+    const repository = body.repository.full_name;
+    const bucketPath = `${uploadDate.getDay()}-${uploadDate.getMonth()}-${uploadDate.getFullYear()}/${repository}/${deliveryId}`; //formats path to DD-MM-YYYY/repository/deliveryid
+
+    const command = new PutObjectCommand({
+      Bucket: process.env.BUCKET_NAME,
+      Key: bucketPath,
+      Body: JSON.stringify(body),
+      ContentType: "application/json",
+    });
+    try {
+      const response = client.send(command);
+      console.log("Successfully uploaded payload to bucket", response);
```

Comment:
Maybe add the delivery ID to these two logging messages just in case we need to debug